pc2cps

```diff
@@ -36,6 +50,7 @@ class pkgconfig(object):
                 if ma is not None:
                     self.attr[ma.group(1)] = self._expand(ma.group(2))
                     continue
+        self._extract_requires()
```

Comment:
Blank line before this, maybe?

I notice your later code is also whitespace impoverished. Please consider adding some blank lines to help separate blocks of code into logical sections.